src/review/routes/build/BuildSummary.js

```diff
@@ -36,12 +34,12 @@ export function BuildSummary(props) {
   )
 }
 
-BuildSummary.propTypes = {
+BuildSummaryView.propTypes = {
   classes: PropTypes.object.isRequired,
   fetch: PropTypes.object.isRequired,
 }
 
 export default recompact.compose(
   withStyles(styleSheet),
   connect(state => state.ui.build),
-)(BuildSummary)
+)(BuildSummaryView)
```

Comment:
There is a rule about import/no-default-export-name" and I think it's important. Also I think we should not export something just for testing. We actually don't test the thing we export, the test has no value for me.